GoogleMapsApi/Entities/Places/Request/PlacesRequest.cs

```diff
@@ -54,12 +53,11 @@ protected internal override string BaseUrl
 		public RankBy RankBy { get; set; }
 
 		/// <summary>
-		/// Restricts the results to Places matching at least one of the specified types. 
-		/// Types should be separated with a pipe symbol (type1|type2|etc). 
+		/// Restricts the results to Places matching the specified type. 
 		/// See the list of supported types - https://developers.google.com/maps/documentation/places/supported_types
 		/// 
 		/// </summary>
-		public string Types { get; set; }
+		public string Type { get; set; }
```

Comment:
Also, I see that you've made the same change in 3 places. Maybe you can refactor and reuse the same property [inheritance?].